templates/api/clients/ruby/client.gemspec.tpl

```diff
@@ -11,11 +11,11 @@ Gem::Specification.new do |spec|
   spec.metadata["source_code_uri"] = spec.homepage
   spec.metadata["github_repo"] = "ssh://github.com/getoutreach/{{ .Config.Name }}"
 
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.7.5")
+  spec.required_ruby_version = Gem::Requirement.new(">= {{ stencil.Arg "versions.grpcClients.ruby" }}")
   spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
   spec.require_paths = ["lib", "lib/{{ .Config.Name }}_client"]
-  spec.add_dependency 'grpc', '~> 1.38'
+  spec.add_dependency 'grpc', '~> 1.59'
```

Comment:
question: should we put this version in the manifest as well since it's referenced in multiple places?

Reply:
That sounds like a reasonable improvement